### DISCUSSION TITLE
Long time no update, are you still in ?

### DISCUSSION THREAD

**Discussion by echolove38** [react-native-elements/react-native-elements#3930] [Q&A] [answered]
if you don't maintance this ui framework, I will change to others

**Comment by ChromeQ** [comment]
Seems like this is dead, what do others recommend as an alternative?

**Reply by echolove38** [reply]
what about react-native-paper

**Reply by ChromeQ** [reply]
I'm already using that as part of react-navigation so I will probably convert over all elements to paper soon enough.

**Comment by theianmay** [comment, marked as answer]
## 🚀 React Native Elements v5 Beta is Live!  

Hello everyone! 👋  

**v5.0.0-beta.1** is now available! This is a **beta release**, intended for testing and feedback. The **full stable v5.0.0** is still in the pipeline.  

---
###⚠️ What you need to know
- This is not yet stable, intended for testing and feedback.
- For example, a known issue is: Button disabled state styling on Android. Workarounds and discussion available here: #3985
- Associated issue: https://github.com/facebook/react-native/issues/53849
- You can install the beta via NPM:
```bash
npm install @rneui/base@beta @rneui/themed@beta
```
---

### 📝 Feedback
Please provide **bugs, suggestions, or feedback**:  
- GitHub Discussion: https://github.com/react-native-elements/react-native-elements/discussions/4007 
- Discord: https://discord.com/invite/e9RBHjkKHa

**Next Steps:**  
- I will be closing this for now to keep things organized.
- All ongoing beta feedback should go to the Discussion or Discord.
